api/src/services/application-csv-export.service.ts

```diff
@@ -313,10 +325,16 @@ export class ApplicationCsvExporterService
       {
         path: 'submissionType',
         label: 'Application Type',
+        format: (val: string): string =>
+          val === ApplicationSubmissionTypeEnum.electronical
+            ? 'electronic'
+            : val,
       },
       {
         path: 'submissionDate',
         label: 'Application Submission Date',
+        format: (val: string): string =>
+          formatLocalDate(val, this.dateFormat, this.timeZone),
```

Comment:
I know its gonna be a pain to track everything down, but I think passing the timezone as an additional argument to the formatters would be a safer bet